server/routes/handlers/book/lastRelease.test.ts

```diff
@@ -1,34 +1,38 @@
 // @ts-nocheck
```

Comment:
Do we no longer need 'addRecallType'?

Reply:
No, it's gone from the wireframes, it'll be calculated automatically based on sentence length